Braze-Demo/SupportingFiles/Local-JSON/Home-List.json

```diff
@@ -0,0 +1,101 @@
+{
+  "pills": [
+    {
+      "title": "Liquid 101",
+      "image": "https://i.picsum.photos/id/866/200/300.jpg?hmac=rcadCENKh4rD6MAp6V_ma-AyWv641M4iiOpe1RyFHeI",
```

Comment:
May be nice to have Braze-hosted images here - to reduce risk of picsum going down or taking away access